sycl/include/sycl/ext/oneapi/matrix/matrix-unified-utils.hpp

```diff
@@ -68,7 +68,7 @@ convertMatrixUseStringToEnum(const char *UseString) {
   return std::nullopt;
 }
 
-inline __SYCL_ALWAYS_INLINE __spv::MatrixLayout joint_matrix_layout_to_spv(
+constexpr inline __spv::MatrixLayout joint_matrix_layout_to_spv(
```

Comment:
FYI: `constexpr` implies `inline`, from [cppreference](https://en.cppreference.com/w/cpp/language/constexpr):

> A `constexpr` specifier used in the first declaration of a function or [static](https://en.cppreference.com/w/cpp/language/static) data member(since C++17) implies `inline`. If any declaration of a function or function template has a `constexpr` specifier, then every declaration must contain that specifier.